coala_langserver/langserver.py

```diff
@@ -110,7 +102,38 @@ def send_diagnostics(self, path, diagnostics):
             'uri': 'file://{0}'.format(path),
             'diagnostics': _diagnostics,
         }
-        self.send_notification('textDocument/publishDiagnostics', params)
+        self._endpoint.notify('textDocument/publishDiagnostics', params=params)
+
+
+@enforce_signature
+def start_tcp_lang_server(handler_class: LangServer, bind_addr, port):
+    # Construct a custom wrapper class around the user's handler_class
+    wrapper_class = type(
+        handler_class.__name__ + 'Handler',
+        (_StreamHandlerWrapper,),
+        {'DELEGATE_CLASS': handler_class, }
```

Comment:
no, a trailing comma is for allowing new additions without modifications to exiting lines.  hence this `,` is not a trailing comma.

`{'DELEGATE_CLASS': handler_class},`